routes/confirmation/checkAnswers.js

```diff
@@ -0,0 +1,138 @@
+const answerInfo = [
+  { 
+    'sectionName': 'Personal Information',
+    'sectionLines': [
```

Comment:
somewhat pedantic, but can we call this 'title' and 'row' or something a bit more terse?